### PR TITLE
Create making-a-resizeable-element.mdx

### PR DESCRIPTION
**Pull request by abbas-roholamin**
## Short Description

Making a resizable element with css resize property

### CODE REVIEW

File: blog/css/making-a-resizeable-element.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Making a Resizable Element
+author: Abbas Roholamin
+category: [css]
+description: .
```
Comment (AliReza1083) [inline blog/css/making-a-resizeable-element.mdx:5]: If you add a short description about your blog, It'd be great.

**Review by AliReza1083 — CHANGES_REQUESTED** [review]
Everything looks good to me.
If you see the preview website, the description is messing, It'd be great to have something there.

File: blog/css/making-a-resizeable-element.mdx (hunk 1 above)
Comment (abbas-roholamin) [reply blog/css/making-a-resizeable-element.mdx]: Description have also been added
you can check

**Review by AliReza1083 — APPROVED** [review]
LGTM 🚀

**Comment by AliReza1083** [comment]
Your hard work is appreciated. 
Thank you for your collaboration. 
We look forward to working with you more in the future.

**Pull request merged**